catalogue/webapp/test/utils/dates.test.ts

```diff
@@ -379,18 +418,29 @@ describe("isRequestableDate: checks the date falls between 2 specified dates and
 
   it('returns true if the there is no start date and the date falls on or before the end date', () => {
     const result = isRequestableDate({
-      date: london('2019-12-20'),
-      endDate: london('2019-12-31'),
+      date: new Date('2019-12-20'),
+      endDate: new Date('2019-12-31'),
+      excludedDates: [],
+      excludedDays: [],
+    });
+    expect(result).toEqual(true);
+  });
+
+  it('returns true if the there is no end date and the date falls after the start date', () => {
+    const result = isRequestableDate({
+      date: new Date('2019-12-20'),
+      startDate: new Date('2019-12-17'),
       excludedDates: [],
       excludedDays: [],
     });
     expect(result).toEqual(true);
   });
 
-  it('returns true if the there is no end date and the date falls on or after the start date', () => {
+  it('returns true if there are start and end dates and the date falls on the start date', () => {
     const result = isRequestableDate({
-      date: london('2019-12-20'),
-      startDate: london('2019-12-17'),
+      date: new Date('2019-12-17T01:00:00Z'),
+      startDate: new Date('2019-12-17T12:00:00Z'),
```

Comment:
This is a new test I added, to catch a bug I introduced during development.

In particular, we care if `startDate < date` or if they're at any time on the same day, but I missed the second condition – adding this boundary test helped me identify the error.